src/api/usersRouter.ts

```diff
@@ -19,27 +19,28 @@ import {
 
 export function createUsersRouter() {
   const usersRouter = express.Router();
-  usersRouter.get('/current', enrichCurrentUser, async (req: UserRequest, res) => {
+  usersRouter.use('/current', enrichCurrentUser);
+  usersRouter.get('/current', async (req: UserRequest, res) => {
     const userWithIsApprover = await enrichUserWithIsApprover(req.user!);
     return res.json(userWithIsApprover);
   });
 
-  usersRouter.put('/current/acceptTerms', enrichCurrentUser, async (req: UserRequest, res) => {
+  usersRouter.put('/current/acceptTerms', async (req: UserRequest, res) => {
     await req.user!.$query().patch({ acceptedTerms: true });
     return res.sendStatus(200);
   });
 
+  usersRouter.get('/current/participant', async (req: UserRequest, res) => {
```

Comment:
It needs exact work for the middleware to be able to match the path with params. For now, this endpoint is only used by get current user's participant. I think it is fine to use `/current` pattern.